app/src/main/java/net/skyscanner/backpack/demo/stories/BarChartStory.kt

```diff
@@ -0,0 +1,61 @@
+package net.skyscanner.backpack.demo.stories
+
+import android.os.Bundle
+import android.view.View
+import net.skyscanner.backpack.barchart.BpkBarChart
+import net.skyscanner.backpack.demo.R
+import java.util.*
+import kotlin.collections.ArrayList
+
+class BarChartStory : Story() {
+
+  override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+    super.onViewCreated(view, savedInstanceState)
+
+    view.findViewById<BpkBarChart>(R.id.bar_chart).apply {
+      invoke(BpkBarChart.Model(
+        groups = listOf(
+          createMonth(0),
+          createMonth(1),
+          createMonth(2),
+          createMonth(3),
+          createMonth(4),
+          createMonth(5)
+        ),
+        legend = BpkBarChart.Legend(
```

Comment:
How does the legend relate to the data in the graph?

Reply:
It shares the same color scheme. The badges are also placed in the way to not affect to tooltips.